index.html

```diff
@@ -1,12 +1,136 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Девайс</title>
-  </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на интенсивном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">Базовый HTML и CSS</a>».</p>
-
-  </body>
+<html lang="ru" xmlns="http://www.w3.org/1999/html">
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Девайс</title>
+</head>
+<body>
+<header class="header">
+  <div class="header1">
+    <ul>
+      <li><button class="h1search">поиск по сайту</button></li>
+      <li><button class="h1enter">войти</button></li>
+      <li><button class="h1compare">сравнить</button></li>
+      <li><button class="h1basket">корзина</button></li>
+    </ul>
+  </div>
+  <div class="header2">
+    <ul><!--задать один стиль всем ссылкам, где ставить class-->
+      <li> <a href="" class="h2">каталог товаров</a></li>
+      <li> <a href="" class="h2">доставка</a></li>
+      <li> <a href="" class="h2">гарантия</a></li>
+      <li> <a href="" class="h2">контакты</a></li>
+    </ul>
+  </div>
+</header>
+<div class="gallery">
+  <!-- не знаю как сделать галерею с слайдами,
+  понимаю, что слева картинка, справа текст
+  и есть 3 кнопки, -->
+  <ul>
+    <li><img src="tovar"> <!-- можно ли заебенить в LI  еще одну UL и имеет ли смысл,
+     или заебенить H1 и далее текст--></li>
+    <li> <img src="tovar2" <h1>Дедлай селфи как бен сттилер</h1> </li>
+  </ul>
+</div>
+<div class="assortiment">
+  <ul>
+    <li><a href=""></a> <img src="вирточки">виртуальная реальность</li>
```

Comment:
Move all content inside `<a>`. Like this:
```suggestion
    <li><a href=""><img src="вирточки">виртуальная реальность</a></li> 
```